tests/components/list/list.spec.js

```diff
@@ -116,6 +120,16 @@ describe("List", () => {
     expect(onClick).toHaveBeenCalledTimes(0);
   });
 
+  test("click will not trigger onClick for disabled item", () => {
+    const onClick = jest.fn();
+    const wrapper = mount(
+      <List width={100} items={itemsWithDisabled} onClick={onClick} />
+    );
+    const itemsWrapper = wrapper.find(Item);
+    itemsWrapper.at(0).simulate("click");
+    expect(onClick).toHaveBeenCalledTimes(0);
```

Comment:
Consider adding more test that the disabled style is applied?

Reply:
@AmirAsaraf it is passed as a prop to rowRenderer which handles styles... renderer wasn't touched